modules/cli/match/match.go

```diff
@@ -61,7 +61,7 @@ func Match(ctx context.Context, opts *Options) error {
 	}
 
 	if opts.Target == "" {
-		m := b.Match(map[string]map[string]string{
+		m := b.Match("", map[string]map[string]string{
```

Comment:
Do we not have the cluster name at this point?

Reply:
This is actually coming from CLI. But yeah, I should add an option in CLI